Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -6130,5 +6144,170 @@ private MediaStream GetMediaStream(IReadOnlyList<IResultSetValue> reader)
 
             return item;
         }
+
+        public List<MediaAttachment> GetMediaAttachments(MediaAttachmentQuery query)
+        {
+            CheckDisposed();
+
+            if (query == null)
+            {
+                throw new ArgumentNullException(nameof(query));
+            }
+
+            var cmdText = "select "
+                        + string.Join(",", _mediaAttachmentSaveColumns)
+                        + " from mediaattachments where"
+                        + " ItemId=@ItemId";
+
+            if (query.Index.HasValue)
+            {
+                cmdText += "AND AttachmentIndex=@AttachmentIndex";
+            }
+
+            cmdText += " order by AttachmentIndex ASC";
+
+            using (var connection = GetConnection(true))
+            {
+                var list = new List<MediaAttachment>();
```

Comment:
maybe move this allocation outside of upper `using()` and put two `using`-s one after another instead of nesting? like

```c#
var list = new List<MediaAttachment>();
using (var connection = GetConnection(true))
using (var statement = PrepareStatement(connection, cmdText))
{
    // do stuff
}
```